primer-api/src/Primer/API/RecordPair.hs

```diff
@@ -1,3 +1,5 @@
+{-# LANGUAGE NoFieldSelectors #-}
+
```

Comment:
Why is this pragma needed?

Reply:
We need to import the type fully in `Primer.API` to use the generic lenses. But `fst` and `snd` as selectors obviously cause a lot of name clashes.